survey-runner-queue/ebs.tf

```diff
@@ -0,0 +1,9 @@
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
```

Comment:
No, good spot, we should also review that. Another option could be to reference the git commit hash for that tag.